packages/components/src/animate/index.js

```diff
@@ -48,6 +48,8 @@ export function getAnimateClassName( options ) {
 			'is-from-' + origin
 		);
 	}
+
+	return undefined;
```

Comment:
⚠️  Runtime change:

Changing return value here from `void` to `undefined`, since we are actually observing the return value of the `getAnimateClassName`. TSC then warned about the missing explicit return statement, which I added